src/sentry/migrations/0590_eventattachment_metadata.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.2.23 on 2023-11-08 11:30
+
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0589_add_commit_date_added_indices"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="eventattachment",
+            name="blob_path",
+            field=models.TextField(null=True),
+        ),
+        migrations.AddField(
+            model_name="eventattachment",
+            name="content_type",
+            field=models.TextField(null=True),
+        ),
+        migrations.AddField(
+            model_name="eventattachment",
+            name="sha1",
+            field=models.CharField(max_length=40, null=True),
+        ),
+        migrations.AddField(
+            model_name="eventattachment",
+            name="size",
+            field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(null=True),
```

Comment:
Should this be a bigint? Would we have files larger than 2,147,483,647 bytes?

Reply:
no, for two reasons:
- the existing `File` model has the same restriction and we do not support files over 2G.
- attachments specifically have even stricter size limits that are already enforced on the relay layer.